### ISSUE TITLE
[CRAN Release]: <v0.9.3>

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/tern#1084]
### PRs

- [x] https://github.com/insightsengineering/tern/pull/1140
- [x] https://github.com/insightsengineering/tern/pull/1143

### Issues

- [x] https://github.com/insightsengineering/tern/issues/1113
- [x] https://github.com/insightsengineering/tern/issues/1062

### Pre-requisites

- [x] Make sure that high priority bugs (label "priority" + "bug") have been resolved before going into the release.
- [x] Review old/hanging PRs before going into the release (Optional).
- [x] Revisit R-package's lifecycle badges (Optional).
- [x] Make sure that all upstream dependencies of this package that need to be submitted to CRAN were accepted before going into release activities.
- [x] Make sure integration tests are green 2-3 days before the release. Look carefully through logs (check for warnings and notes).
- [x] Decide what gets merged in before starting release activities.

### Release Checklist

- [x] Update NEWS.md file: make sure it reflects a holistic summary of what has changed in the package.
- [x] Remove the additional fields (`Remotes` and `Config/Needs/*`) from the DESCRIPTION file where applicable.
- [x] Increase versioned dependency on {package name} to >=X.X.X (Optional).
- [x] Make sure that the minimum dependency versions are updated in the DESCRIPTION file for the package and its reverse dependencies (Optional).
- [x] Create a pull request to make necessary bug fixes/changes (add "[skip vbump]" in the pr title), and after merging the PR, tag the update(s) as a release candidate v < intended release version > -rc < release candidate iteration > on the main branch.
- [x] Build the package locally using the command:`R CMD build .` which will generate a .tar.gz file necessary for the CRAN submission.
- [x] Submit the package that was build in the previous step via this form: https://cran.r-project.org/submit.html.
- [x] Address CRAN feedback, tag the package vX.X.X-rc(n+1) and repeat the submission to CRAN whenever necessary.
- [x] Get the package accepted and published on CRAN.
- [x] If the additional fields were removed, add them back in a separate PR, and then merge the PR back to main. Note: Take precautionary measures to ensure that the version bump does not take place on a merge.
- [x] Create a git tag with the final version set to X.X.X on the main branch.

### Testing

- [x] Integration tests results - accepted.
- [x] UAT results - accepted.
- [x] All testing activities are finalized.

### Release Feedback

- [x] Fix 1
- [x] Enhancement 1
- [x] Defect 1

### Post-release Checklist

- [ ] Make sure that the package is published to internal repositories.
- [x] Review and update installation instructions for the package wherever needed (Optional).
- [ ] Update all integration tests to reference the new release.
- [x] Ensure a new dev version (.9XXX) is added to the NEWS.md file and DESCRIPTION file as a placeholder for release notes.
- [x] Announce the release on ________.

### Decision tree

Click [here](https://github.com/insightsengineering/.github/blob/main/.github/ISSUE_TEMPLATE/RELEASE_DECISION_TREE.md) to see the release decision tree.
